src/globus_cli/login_manager/scopes.py

```diff
@@ -125,4 +118,4 @@ def get_by_resource_server(self, rs_name: str) -> _ServiceRequirement:
 # version we were at when we got a token
 # it should be the max of the version numbers required by the various different
 # services
-CURRENT_SCOPE_CONTRACT_VERSION: t.Final[int] = 1
+CURRENT_SCOPE_CONTRACT_VERSION: t.Final[int] = 2
```

Comment:
The version bump wasn't outright required for the changes to function correctly, but I decided to go with it since I felt the spirit of the story was making sure we had the “correct” solution. So even though the users' old tokens would work, they should still be forced to make the new “correct” ones.